packages/cli/templates/react/index.html

```diff
@@ -9,7 +9,7 @@
     <title>Storyblok Field Plugin</title>
   </head>
   <body>
-    <div id="plugin"></div>
+    <div id="app"></div>
```

Comment:
We're using `#app` but here it was `#plugin`.

(It is still fine though because inside `main.ts` we create `#app` if it doesn't exist)

Reply:
Perhaps it's best to omit it altogether? Because the developer could feel tempted to remove the `document.createElement` as it seems redundant. In production however, they will receive an unpleasant surprise...

Reply:
Oh I didn't see this comment. So you're saying we should remove that #app from the html files, right?

Reply:
Yes exactly :)